src/microbe_stage/Microbe.cs

```diff
@@ -1240,24 +1249,26 @@ public override void _Process(float delta)
         HandleHitpointsRegeneration(delta);
         HandleReproduction(delta);
 
+        HandleOsmoregulation(delta);
+
+        // Let organelles do stuff (this for example gets the movement force from flagella)
+        foreach (var organelle in organelles.Organelles)
+        {
+            organelle.Update(delta);
+        }
+
         // Handles engulfing related stuff as well as modifies the
         // movement factor. This needs to be done before Update is
         // called on organelles as movement organelles will use
         // MovementFactor.
         HandleEngulfing(delta);
+        HandleSprint(delta);
+        HandleStress(delta);
 
         // Handles binding related stuff
         HandleBinding(delta);
         HandleUnbinding();
 
-        HandleOsmoregulation(delta);
-
-        // Let organelles do stuff (this for example gets the movement force from flagella)
-        foreach (var organelle in organelles.Organelles)
-        {
-            organelle.Update(delta);
-        }
-
```

Comment:
I moved these lines so that the movement changes of engulfing and sprinting are applied after the effects of organelles, so that those bonuses are affected by states.